covid-vaccine-tracker/covid-vaccine-tracker.js

```diff
@@ -0,0 +1,298 @@
+const  icon = document.getElementById("icon");
+icon.onclick = function(){
+    document.body.classList.toggle("dark-mode");
+    if ( document.body.classList.contains("dark-mode")) {
+        icon.src="../assets/sun.svg";
+    }else{
+        icon.src="../assets/dark-mode.png";
+    }
+}
+
+document.getElementById("back").onclick = function () {
+  location.href = "../home/home.html";
+}
+//popup
+const button = document.querySelector("#Symptoms-button");
+const myCircule = document.getElementById("myCircule")
+button.addEventListener("click", move);
+function move() {
+  window.location.href = "../Symptoms/Symptoms.html";
+}
+const openModalButtons = document.querySelectorAll("[data-modal-target]");
+const closeModalButtons = document.querySelectorAll("[data-close-button]");
+const overlay = document.getElementById("overlay");
+
+openModalButtons.forEach((button) => {
+  button.addEventListener("click", () => {
+    const modal = document.querySelector(button.dataset.modalTarget);
+    openModal(modal);
+    myCircule.style.setProperty('stroke-dasharray',( (156-156*totalPerHundred).toString()+" "+(156*totalPerHundred).toString()))
```

Comment:
As I mentioned above its a bug if totalPerHundred >1

imagine that totalPerHundred = 1.1 then (156-156*totalPerHundred).toString() will be "-15.6" and thats an invalid value to set as style property for stroke-dasharray so you will get wrong chart